extensions/quarto-document/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "version": 1,
+  "locale": "en_US",
+  "platform": "4.4.1",
+  "metadata": {
+    "appmode": "quarto-static",
+    "primary_rmd": "index.qmd",
+    "primary_html": null,
+    "content_category": null,
+    "has_parameters": false
+  },
+  "quarto": {
+    "version": "1.7.13",
+    "engines": ["markdown"]
+  },
+  "packages": null,
+  "files": {
+    "_quarto.yml": {
+      "checksum": "e498dcc5956dba39a6df10ac9568fcb9"
+    },
+    "index.qmd": {
+      "checksum": "52f82c15783f37b2de073e22e1b5a648"
+    },
+    "thumbnail.jpg": {
+      "checksum": "3691931bc6c8ff0b84317140edbd4b8b"
+    }
```

Comment:
The `thumbnail.jpg` isn't present here so we probably want to remove that from the `manifest.json`